pennylane/devices/default_mixed.py

```diff
@@ -544,7 +588,10 @@ def _apply_operation(self, operation):
         if operation in diagonal_in_z_basis:
             self._apply_diagonal_unitary(matrices, wires)
         else:
-            self._apply_channel(matrices, wires)
+            if len(wires) > 1:
```

Comment:
Is the einsum implementation much faster for one qubit? Otherwise I would say that we could simply replace it.

Reply:
I think we should maybe leave it in, there is a noticeable difference. Of course these are small numbers, but the relative difference is not small:
![Screenshot from 2023-01-09 09-37-38](https://user-images.githubusercontent.com/20772922/211268061-3a7e7494-5103-4f18-9ba6-5dab6446340f.png)

This is a timing of applying `RX(0.5, 0)` to a `n`-qubit  register. 
I also checked applying `RX(0.5, n-1)` (last instead of first wire), which resulted in:

![Uploading Screenshot from 2023-01-09 09-40-57.png…]()